lib/membrane/video_compositor/events.ex

```diff
@@ -0,0 +1,59 @@
+defmodule Membrane.LiveCompositor.Events do
+  @moduledoc """
+  Events sent to the parent process
+  """
+
+  alias Membrane.LiveCompositor.Context
+  alias Membrane.Pad
+
+  @typedoc """
+  The output pad was connected and the TCP connection was established between pipeline and
+  the LiveCompositor server.
+  """
+  @type output_registered_event_type :: :output_registered
+
+  @typedoc """
+  The input pad was connected and the TCP connection was established between pipeline and
+  the LiveCompositor server.
+  """
+  @type input_registered_event_type :: :input_registered
+
+  @typedoc """
+  The compositor instance received the input, and the first frames/samples of that input are
+  ready to be used.
```

Comment:
Are they? What about buffering?

Reply:
responded already in `video_compositor` repo